### ISSUE TITLE
Should match exact text to click by default

### ISSUE THREAD

**Issue by sguptatw** [getgauge/taiko#40]
**Actual behavior**
By default if the text starts with the given text, the element is clicked.

**Steps to replicate**
* Create a directory test
`git clone https://github.com/getgauge/taiko.git`
* Checkout branch taiko-cri
`git checkout taiko-cri`
* install taiko
npm install <fullPath> -g
* Create a testDirectory
Add a exactClick.js file with 
```
const { browser, openBrowser, goto, link, click, listItem } = require('taiko');

(async () => {
    try {
        await openBrowser();
        await goto("http://google.com");
        await click('Gmai')
    } catch (e) {
        console.error(e);
    } finally {
        if(browser())
            closeBrowser();
    }
})();
```
* run this `taiko exactClick.js`
> The element Gmail is clicked even though the text given was Gmai
The default behavior should be exact match. It should be configurable.

**Version**
```
Version: 0.0.9 (HeadlessChrome/69.0.3452.0)
```
Blocks #29

**Comment by Apoorva-GA** [comment]
Changed the default behaviour select the exact match.
However we can configure the click to not match exactly using `click('Gmai', {exactMatch : false})`.
Fix should be available in commit cf8bb5d88e76a57f0dc19f7292d747114056667c

**Comment by Apoorva-GA** [comment]
`click('Gmail')` Will look for the exactMatch.
 If we want to click on element containing text `Gmai`, `click(contains('Gmai'))` should work.
